.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: tests
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # Help Codecov detect commit SHA
+          fetch-depth: 2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 15
```

Comment:
Let's just go all the way to node 16 since this is a new environment. Unless something doesn't work, then it may be easier to just leave it for now.

Reply:
Yeah 16 breaks things haha

Reply:
`node-sass` seems to need an upgrade for every node version 😢 It's been a pain.

Thankfully migrating to dart-sass (npm `sass` package) no longer needs a native build step.
https://github.com/sass/node-sass#node-sass
> Warning: LibSass and Node Sass are deprecated. While they will continue to receive maintenance releases indefinitely, there are no plans to add additional features or compatibility with any new CSS or Sass features. Projects that still use it should move onto Dart Sass.

I've migrated a couple repos:
https://github.com/CruGlobal/mpdx_web/pull/2232 https://github.com/CruGlobal/mpdx_web/pull/2235
https://github.com/CruGlobal/campus-contacts-web/pull/837

Doesn't need to happen in this PR but that's how to do it :) Hopefully we don't have to fix too many things but I have no idea what the state of the sass is in this project. It should spit out warnings for everything.

I used
```js
                    {
                        loader: 'sass-loader',
                        options: {
                            sassOptions: {
                                quietDeps: true,
                            },
                        },
                    },
```
to swallow warnings in NPM packages that had outdated sass files.